verifier/src/main/java/com/okta/jwt/JwtHelper.java

```diff
@@ -36,7 +36,8 @@
 public final class JwtHelper {
 
     private String issuerUrl;
-    private String clientOrAudience;
+    private String audience = "api://default";
```

Comment:
a user could call `setAudience(null)`, the default just allows them to not need to call the setter